src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/alerts/AlertMappingService.kt

```diff
@@ -92,6 +97,22 @@ class AlertMappingService(
     )
   }
 
+  suspend fun getMappings(pageRequest: Pageable): Page<AlertMappingDto> = coroutineScope {
+    val mappings = repository.findAllBy(
+      pageRequest = pageRequest,
+    )
+
+    val count = async {
+      repository.count()
+    }
```

Comment:
Why just 1 async call? Not sure how this helps?

Reply:
Well the `mappings.toList()`  is the 2nd hidden one (I think!)

Reply:
But actually, yes I think you right, I will do another PR to fix